### DISCUSSION TITLE
GPT-4 Turbo

### DISCUSSION THREAD

**Discussion by ZYinMD** [ChatGPTNextWeb/NextChat#3169] [General]
OpenAI's accouncement today:
> ...GPT-4 Turbo, our most advanced model... offers a 128K context window and knowledge of world events up to April 2023...

Is this being added in the app? The updated training knowledge up to April 2023 sounds really really important.

**Comment by H0llyW00dzZ** [comment]
yes already added in latest version

**Reply by ZYinMD** [reply]
I tested `gpt-4-1106-preview`, and it indeed has knowledge up to April 2023, which is really nice!. But I don't see `gpt-4-turbo` yet.

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/32368482/cbd660e1-3a87-4a28-8914-1ff7f07beb0c)

**Reply by H0llyW00dzZ** [reply]
gpt-4 turbo is that `1160` preview and vision
